python-avd/pyavd/_utils/password_utils/password_utils.py

```diff
@@ -11,8 +11,6 @@
 
 import base64
 
-from ansible_collections.arista.avd.plugins.plugin_utils.errors import AristaAvdError
-
 try:
```

Comment:
We don't want the try/except for imports in pyavd. Also remove all the `if not HAS_CRYPTOGRAPHY` below. It is a special workaround needed for Ansible.